src/Repository/EventRepository.php

```diff
@@ -131,9 +134,7 @@ protected function createSlugQueryBuilder(string $slug): QueryBuilder
      */
     public function findManagedBy(Adherent $referent): array
     {
-        if (!$referent->isReferent()) {
-            return [];
-        }
+        $this->checkReferent($referent);
```

Comment:
not the same thing, `return []` vs `throw new Exception`

Reply:
Yes, but in this case it's better to throw an exception because this method should not be called with non referent.